cbmackintosh/whats-cookin
# Feature/carousel

## Is this a fix or a feature?
This is a feature
## What is the change?
* Carousel has been added to the main page
## What does it fix?
n/a
## Where should the reviewer start?
loadCarousel function has been changed to work with carousel classes review that function
Take a look over the glide.js script 
Effects are listed on the glide.js website if you would like to see all the options.
## How should this be tested?
load the home page and click the arrows to move to recipe cards.
*  on click they should move
*  the carousel should auto play
*  the carousel should stop on hover
## Relevant ScreenShots
![Feb-13-2021 16-50-49](https://user-images.githubusercontent.com/72312636/107864749-b1705200-6e1c-11eb-8f61-286ea1c609d0.gif)
